### DISCUSSION TITLE
Successfully updated form ventura 13.7.1 to MacOS 15.1.1 Sequoia

### DISCUSSION THREAD

**Discussion by cyrhex** [ChefKissInc/NootedRed#336] [Show and tell]
the only issue i've encounter are sometimes about this mac doesnt launch and system settings is crashing,but overall performance is good,i cud run imovie and final cut pro
![Screenshot 2024-12-05 at 11 20 15 AM](https://github.com/user-attachments/assets/922699f0-f96b-49b3-a9bb-5fd7ea9fb470)

**Comment by lwh110221** [comment]
🎉

**Comment by AlfCraft07** [comment]
> the only issue i've encounter are sometimes about this mac doesnt launch and system settings is crashing

I also have this issue, even when I had Ventura.

**Reply by VisualEhrmanntraut** [reply]
I think this issue happens to every Hackintosh on new macOS versions. My best guess, based on the crash log, is that macOS hates the user space patching, e.g. by RestrictEvents.

**Reply by cyrhex** [reply]
> I think this issue happens to every Hackintosh on new macOS versions. My best guess, based on the crash log, is that macOS hates the user space patching, e.g. by RestrictEvents.

very stable now using the latest commit,no crashing at all,,nice work
